lib/test-worker.js

```diff
@@ -27,6 +27,9 @@ globals.options = opts;
 
 const serializeError = require('./serialize-error');
 
+// Initialize color support
+chalk.enabled = typeof opts.color === 'boolean' && opts.color;
```

Comment:
`opts.color` already is a boolean, courtesy of https://github.com/avajs/ava/blob/1df502df8cba18f92afcbaed730d8c724687ca45/lib/cli.js#L78.

Reply:
Yeah, I did this just to make [this](https://github.com/avajs/ava/blob/master/test/fork.js#L144) test pass since it doesn't run the CLI.

Reply:
That's an odd test. Just remove it 😄